packages/tracer/src/config/EnvironmentVariablesService.ts

```diff
@@ -40,7 +44,7 @@ class EnvironmentVariablesService extends ConfigService {
   }
 
   public getXrayTraceId(): string {
-    return this.get(this.xRayTraceIdVariable);
+    return super.getXrayTraceId();
```

Comment:
Same as for `EnvironmentVariablesService` in logger, are we sure we need this?

Reply:
Thanks, removed, see here https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1123#discussion_r999231243